audio.py

```diff
@@ -30,9 +35,32 @@ def read_audio(
 		elif audio_path.endswith('.wav'):
 			sample_rate_, signal = scipy.io.wavfile.read(audio_path)
 			signal = torch.as_tensor(signal[None, :] if len(signal.shape) == 1 else signal.T)
+		elif audio_path.endswith('.gsm'):
```

Comment:
Please do a `backend == 'soundfile'` check instead. While we're still studying perf, it may be good to be able to run ffmpeg backend on gsm files as well for comparison